pennylane/init.py

```diff
@@ -161,9 +161,14 @@ def random_layers_uniform(n_layers, n_wires, n_rots=None, low=0, high=2 * pi, se
     if seed is not None:
         np.random.seed(seed)
 
+    # set default
     if n_rots is None:
         n_rots = n_wires
 
+    # no circuit if there are no wires
+    if n_wires == 0:
```

Comment:
When did these cases arise? (perhaps I had this question before :smile:) Wondering if it's a good call to "silently" return an empty array as the output of `np.random.uniform`. Would a warning work here or that would be too much?

Reply:
I know, you are probably right :) I thought a user may iterate over wires and start from 0, but then the qnode would complain...

Maybe it would be better in future to throw an error - will put this on the todo list for the templates upgrade!